docs/root/start/sandboxes/gzip.rst

```diff
@@ -85,8 +85,10 @@ Now, use ``curl`` to make a request for the compressed statistics:
     content-encoding: gzip
 
 .. seealso::
+   :ref:`Gzip Api <envoy_v3_api_field_extensions.filters.http.gzip.v3.Gzip.compressor>`
+      Learn more abort Envoy’s gzip Apis.
```

Comment:
and this one i think something like "API and configuration reference for Envoy's gzip compression"